examples/step-78/doc/intro.dox

```diff
@@ -0,0 +1,213 @@
+    &-\frac{\partial V}{\partial \tau} + \frac{\sigma^2S^2}{2}\frac{\partial^2 V}{\partial S^2} + rS\frac{\partial V}{\partial S} - rV = 0, \
+    \quad\quad &&\forall S\in [0,S_\text{max}], \tau \in [0,T]
+    \\
+    &V(0,\tau) = 0, \
+    &&\forall \tau \in [0,T]
+    \\
+    &V(S_\text{max},\tau)=S_\text{max}-Ke^{-r\tau}, \
+    &&\forall \tau \in [0,T]
+    \\
+    &V(S,0) = \max(S-K,0) \
+    &&\forall S \in [0,S_\text{max}]
+\end{align*}
+
+Conceptually, this is an advection-diffusion-reaction problem for the variable $V$: There is both a second-order derivative diffusion term, a first-order derivative advection term, and a zeroth-order reaction term.
+We can expect this problem to be a little bit forgiving in practice because for realistic values of the coefficients, it is diffusive dominated. But, because of the advective terms in the problem, we will have to be careful with mesh refinement and time step choice. There is also the issue that the diffusion term is written in a non-conservative form and so integration by parts is not immediately obvious.
+
+We will solve this problem using the fractional step method (of which the Crank-Nicolson method is a special case with $\theta=\frac 12$; the explicit Euler method corresponds to $\theta=0$ and the implicit Euler method to $\theta=1$). So, we first discretize in time, where we would like $V^n(S)$ to approximate $V(S,\tau_n)$:
+\begin{align*}
+    0=&-\frac{V^n(S)-V^{n-1}(S)}{k_n} \\
+    &+\frac{\sigma^2S^2}{2}\left[(1-\theta)\frac{d^2V^{n-1}(S)}{dS^2} + \theta \frac{d^2V^{n}(S)}{dS^2}\right] \\
+    &+rS\left[(1-\theta)\frac{dV^{n-1}(S)}{dS} + \theta\frac{dV^{n}(S)}{dS}\right]  \\
+    &-r\left[(1-\theta)V^{n-1}(S) + \theta V^n(S)\right]
+\end{align*}
+Here, $k_n=\tau_n-\tau_{n-1}$ is the time step size. Given this time discretization, we can proceed to discretize space by multiplying with test functions and then integrating by parts. Because there are some interesting details in this due to the advective and non-advective terms in this equation, this process will be explained in detail.
+
+So, we begin by multiplying by test functions, $\{\phi_i(S)\}_{i\in\mathbb{N}}$:
+\begin{align}
+    0=&-\int_0^{\smax}\phi_i(S)\left[V^n(S)-V^{n-1}(S)\right]dS \\
+    &+k_n\int_0^{\smax}\phi_i(S)\left[\frac{\sigma^2S^2}{2}\left[(1-\theta)\frac{d^2V^{n-1}(S)}{dS^2} + \theta \frac{d^2V^{n}(S)}{dS^2}\right]\right]dS \\
+    &+k_n\int_0^{\smax}\phi_i(S)\left[rS\left[(1-\theta)\frac{dV^{n-1}(S)}{dS} + \theta\frac{dV^{n}(S)}{dS}\right]\right]dS  \\
+    &-k_n\int_0^{\smax}\phi_i(S)\left[r\left[(1-\theta)V^{n-1}(S) + \theta V^n(S)\right]\right]dS
+\end{align}
+
+
+As usual, (1) becomes $-\bold{M}V^n+\bold{M}V^{n-1}$ and (4) becomes $k_n\left[-r(1-\theta)\bold{M}V^{n-1} - \theta r\bold{M}V^n\right]$, where $\bold{M}_{i,j}=\left<\phi_i(S),\phi_j(S)\right>$, and where we have taken the liberty of denoting by $V$ not only the function $V(S)$ but also the vector of nodal values after discretization.
```

Comment:
Why not round brackets in `$\bold{M}_{i,j}=\left<\phi_i(S),\phi_j(S)\right>$`?

Reply:
I changed this and all the other places that this happened to match the other tutorials.